content/packit-as-a-service.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Packit-as-a-Service"
+date: 2019-06-22
+draft: false
+---
+
+## Introductory description
+With packit, it's easy to integrate your upstream project into Fedora Project.
+Your pull requests are built in Fedora version of your choice,
+so you're always sure that your software works. When you're ready for a release,
+packit takes care of all the hassle and is able to ship your new upstream release
+into Fedora Rawhide.
+
+### Easy to use
+
+Using Packit Service is very straightforward: add one config file to your repository,
+followed by an RPM spec file and you're good to go. We also support packit as a CLI tool,
+so you can always try things locally on your own.
+
+### Makes you confident
+
+Packit Service validates your pull requests by building your software in Fedora OS.
+Once the builds are done, packit lets you know how to install your change
+inside your environment. Get confidence by trying out your changes yourself
+before shipping them to your users.
```

Comment:
I'm not sure we need a marketing pitch in the intro/docs, how about putting it to "About"?

Reply:
Yeah, it could be. I will move it.

Reply:
Does it make sense to create in "About" section two subsections like:

~~~
$ About.md
---
title:
---
## Packit-as-a-Service
marketing

## Packit
info about packit
~~~

Reply:
I wouldn't pollute "About" with this.
I think Tomas just wanted to say that we don't need such marketing tone.
What I'd suggest:
- remove `### Makes you confident` to merge this with the previous section
- remove the `Get confidence` and merge the sentences into `Once the builds are done, packit lets you know how to install the RPMs inside your environment so you can try the changes before merging them.`

Reply:
ok, I would prefer to remove it as well completely. Waiting for other opinions.

Reply:
> Waiting for other opinions.

there's no one else, we are all alone here :-)

Reply:
Like @TomasTomecek. He already reacts by emoji.